### ISSUE TITLE
Components#getActionExpressionsAndListeners returns actionListener's FQCN

### ISSUE THREAD

**Issue by VsevolodGolovanov** [omnifaces/omnifaces#650]
Version 3.11.2.

```
<p:commandButton value="Download" actionListener="#{bean.download}" ajax="false" process="@this" />
```
This causes FacesRequestLogger to log `javax.faces.event.MethodExpressionActionListener` which is not very interesting. I get that ActionListener could be a custom subclass, but there is no custom class here - the expression is the most interesting part here, but it's not logged. This is probably the most common kind of usage of actionListener, since PrimeFaces for whatever reason promotes it in its docs and showcases.

So it would be great if Components#getActionExpressionsAndListeners would return the expression instead of FQCN in such cases.

**Comment by BalusC** [comment]
Agreed and fixed. I've reused and improved the existing getField() trick which was used for BehaviorListener.

**Comment by VsevolodGolovanov** [comment]
@BalusC, wrong issue link in What's new instead of this one:
![image](https://user-images.githubusercontent.com/8070317/150503648-11b469b9-8b10-40fc-b7c3-9f3a6810febf.png)

**Comment by BalusC** [comment]
Fixed for 3.13 showcase. Thanks!
